src/test/java/runner/TestRunner.java

```diff
@@ -19,6 +22,13 @@
 )
 
 public class TestRunner {
+	
+		@BeforeClass
+		public static void cleanLocustChartsDirectory() throws IOException {
+			//Delete and create the locustcharts folder in order to ensure that exists in every execution
```

Comment:
Like this if you have multiple executions, you will not have saved the files of the previous reports.

Reply:
If I'm not wrong, in a multiple executions, probably you will have differents targets, in'n it? I think that you never launch the same runner at the same time ... and if you launch the same runner probably from a different pipeline, and this will generate another target directory ...

Reply:
Without these 2 lines, the code wouldn't do the same?

Reply:
No. With this two lines, we ensure that if the folder is not created by the user, the system will do it (in other way it will fail), and in case that exists will remove the old pictures from previous executions.

Reply:
If your dynamic folder is timestamped or something, you can save yourself the pain of having this two extra lines here in the hooks

Reply:
hey, then we will need to make the report dinamic too ... Anyway, this extra lines ensure that if the chart folder it not exist, it is created ... maybe we should discuss this in the stand up.